### PR TITLE
My work & about me section.

### PR DESCRIPTION
**Pull request by Taher-web-dev**
In this branch, I hade worked on my work section and about me section of my mobile portfolio app. So  I had prepared a html file named template2.html and a css file named template2.css, which oraganise theses sections. Thus you can check them and send me your feedback before to merge with master branch.

### CODE REVIEW

**Review by TedLivist — COMMENTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST ❌ ❌ 

Hi @Taher-web-dev,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, which failed the following criteria:
- Pull Request should have a descriptive title telling you what changes are inside.
- Pull Request should have a short summary describing introduced changes in more detail.

taher


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: template2.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+               <p>css</p>
+               <p>javaScript</p>
+             </div>
+             <div class="external_link">
+                 <a href="#">See Project</a>
+             </div>
+            </footer>
+
+          
+        </section>
```
Comment (jmontanholi) [inline template2.html:41]: - [ ] All your works should be inside one section and this section should use a `display: grid;`. Also, please fix the margins as of now your project works are very spaced apart.
image

File: template2.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
+<!DOCTYPE html>
```
Comment (jmontanholi) [inline template2.html:1]: # [OPTIONAL]
- [ ] I see that you did all sections but you did it apart from the original mobile work (the header and the hello section). Consider putting all on the same HTML page.

File: template2.html (hunk 1 above)
Comment (jmontanholi) [inline template2.html:41]: - [ ] The content of the projects should be aligned using flexbox, please consider changing it.

File: template2.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+               <p class="project_description">
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+               </p>
+          </div>
+          <footer>
+             <div id="skills_0" class="skills">
+               <p>html</p>
+               <p>css</p>
+               <p>javaScript</p>
+             </div>
```
Comment (jmontanholi) [inline template2.html:34]: - [ ] The list of technologies should be using `ul` tag with `li` tags inside, please consider changing it. (ul stands for an unordered list and li stands for list item). This way your code will be more semantic.👍🏻

File: template2.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+                      <p>Frameworks</p>
+                      <div><img src="statics/images/select.png" alt="select"></div>
+                  </div>
+                  <div id="l1"> <hr/></div>
+                  <div id="sk">
+                    <p>Skills</p>
+                    <div><img src="statics/images/select.png" alt="select"></div>
+                  </div>
+                  <div class="ligne" id="l2"> <hr/></div>
+                </div>
```
Comment (jmontanholi) [inline template2.html:178]: - [ ] Languages, frameworks, and skills should also be `li` tags inside a `ul` tag, please consider changing this. This way your code will be more semantic.

File: template2.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+                    <li><a href="#"><img src="statics/images/doits.png" alt="doit"></a></li>
+                    <li><a href="#"><img src="statics/images/twitter.png" alt="twitter profile link"></a></li>
+                    <li><a href="#"><img src="statics/images/M.png" alt="M link"></a></li>
+                </ul>
+                <a id="my_resume" href="#">Get my resume</a> 
+              </div>
+              <div id="my_skills">
+                  <div id="languages_skills"> 
+                      <div id="spec_name">
+                          <p>Langguages</p>
```
Comment (jmontanholi) [inline template2.html:150]: - [ ] Here is a typo, please consider fixing it in order to have a more professional project.

File: template2.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+                 <p>javaScript</p>
+               </div>
+               <div class="external_link">
+                   <a href="#">See Project</a>
+               </div>
+              </footer>
+  
+            
+          </section>
+          <section id="about_me">
```
Comment (jmontanholi) [inline template2.html:131]: - [ ] The about section should have its content aligned using flexbox instead of grid. Please consider changing it.

Comment (jmontanholi) [inline template2.html:131]: - [ ] In the Figma design this section has just a bit more of the left margin and it is more centered.

Yours: 
image
Figma:
image

**Review by jmontanholi — CHANGES_REQUESTED** [review]
Hello Taher Haggui! 

You have done a great job so far, but there are still some things that you can improve before going to the next project. 😄 

✅ Your project is looking very similar to the design. Good job!
☑️ You are using fonts similar to the ones in the design.
✅ Your README is looking great!
## Required Changes ♻️
_Check the comments under the review._

- [ ] You have some typos in the README, please consider fixing it, this way the README will be more professional. 👍🏻 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, working on optional suggestions is always good, with them you may learn something new or make your project look more professional._

You can also consider:
- [ ] You can deploy the project and put a live demo link in the README, this will make it more professional.
- [ ] You could put a screenshot of your project on the README, this will make it more professional.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by safafa — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUESTED ♻️ 

Hi @taherhagui , Good job working on some of the changes requested by the previous reviewer 👏🏼 👏🏼 

But,  there are still some which aren't addressed yet.

## Required changes :

Check the inline comments.

## Optional changes :
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mke2111 — APPROVED** [review]
### Hi @Taher-web-dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**